packages/blade/src/components/ActionList/ActionListItem.tsx

```diff
@@ -306,8 +312,20 @@ const _ActionListItem = (props: ActionListItemProps): JSX.Element => {
   const isMobile = platform === 'onMobile';
 
   const renderOnWebAs = props.href ? 'a' : 'button';
-  const isSelected =
-    typeof props._index === 'number' ? selectedIndices.includes(props._index) : undefined;
+
+  const getIsSelected = (): boolean | undefined => {
```

Comment:
Can you add some comments here what this is doing?

Reply:
done